content/slides/intro-to-jupyterlab.qmd

```diff
@@ -35,47 +35,78 @@ JupyterLab is not shared!
 
 👪 JupyterLab is a human-friendly interface to computation resources.
 
+![Jupyterlab logo](/_media/jupyterlab_about_logo.png)
+
+:::{.notes}
+The jupyterlab logo is a screenshot from Jupyterlab's "Help -> About JupyterLab"
+popup in cryocloud.
+:::
+
+#
+
+![Jupyterlab](/_media/jupyterlab.png)
+
 
 ## File browser
+:::::: {.columns}
 
+::: {.column}
 * Drag & drop from your desktop
 * Open many types of files and get a useful display
     * CSV will display as table
     * GeoJSON will display on a map
     * Images will open with an image viewer
+:::
 
-* _TODO: Image of file browser on the side of this slide_
+::: {.column}
+![Jupyterlab file browser](/_media/jupyter_lab_file_browser.png)
+:::
 
+::::::
 
 ## Terminal
```

Comment:
I think it'd be really nice to include an image of the app icon on these slides!
![image](https://user-images.githubusercontent.com/18539827/229851611-663ead20-f669-49c4-9143-6320c8727afc.png)